frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -351,6 +352,7 @@ export default class ProblemDetails extends Vue {
   @Prop({ default: null }) problemAlias!: null | string;
   @Prop() isAdmin!: boolean;
   @Prop({ default: false }) showVisibilityIndicators!: boolean;
+  @Prop({ default: () => new Date() }) nextSubmissionTimestamp!: Date;
```

Comment:
```suggestion
  @Prop({ default: () => new Date() }) nextSubmissionTimestamp!: null | Date;
```

los tipos de TypeScript indican que esto puede ser nulo, así que hay que declarar esto consistentemente. pero ahora en frontend/www/js/omegaup/components/arena/RunSubmitPopup.vue también hay que declarar eso como que puede recibir nulo y ajustar el código para que no arroje excepciones cuando es nulo.